### DISCUSSION TITLE
Why is highlighting not working for my parser?

### DISCUSSION THREAD

**Discussion by LaBatata101** [nvim-treesitter/nvim-treesitter#6719] [Q&A]
I've created this [parser](https://github.com/LaBatata101/tree-sitter-bend) and I'm trying to use in Neovim but the highlight is not working.

I'm using this config in neovim:
```lua
local parser_config = require("nvim-treesitter.parsers").get_parser_configs()
parser_config.bend = {
  install_info = {
    url = "~/Code/tree-sitter-bend", -- local path or git repo
    files = { "src/parser.c", "src/scanner.c" }, -- note that some parsers also require src/scanner.c or src/scanner.cc
    -- optional entries:
    branch = "main", -- default branch in case of git repo if different from master
  },
}

vim.filetype.add({
  extension = {
    bend = "bend",
  },
})

vim.treesitter.language.register("bend", { "bend" })
```

**Comment by clason** [comment]
You need to actually enable highlighting with your parser.

**Reply by clason** [reply]
And install it, of course; neither of which happens in the snippets you shared.

**Reply by LaBatata101** [reply]
How can I do that?

**Reply by LaBatata101** [reply]
> And install it, of course; neither of which happens in the snippets you shared.

I'm installing manually with `TSInstall bend`
